From 9e29fee73a3c62b762cbdaa10d0a9493d3b36d7b Mon Sep 17 00:00:00 2001
From: Alex Burmashev <alexander.burmashev@oracle.com>
Date: Tue, 20 Aug 2024 08:57:17 +0000
Subject: [PATCH] test

---
 SPECS/libreoffice.spec | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/SPECS/libreoffice.spec b/SPECS/libreoffice.spec
index a879edf..ddb11ef 100644
--- a/SPECS/libreoffice.spec
+++ b/SPECS/libreoffice.spec
@@ -8,7 +8,7 @@
 %global libo_buildfix %{nil}
 # rhbz#715152 state vendor
 %if 0%{?rhel}
-%global vendoroption --with-vendor="Red Hat, Inc."
+%global vendoroption --with-vendor="OpenELA Enterprise Software Foundation"
 %endif
 %if 0%{?fedora}
 %global vendoroption --with-vendor="The Fedora Project"
@@ -1037,17 +1037,6 @@ rm -rf git-hooks */git-hooks
 %global __scm git_am
 %__scm_setup_git_am
 
-#Customize Palette to add Red Hat colours
-(head -n -1 extras/source/palettes/standard.soc && \
- echo -e '  <draw:color draw:name="Red Hat 1" draw:color="#cc0000"/>
-  <draw:color draw:name="Red Hat 2" draw:color="#0093d9"/>
-  <draw:color draw:name="Red Hat 3" draw:color="#ff8d00"/>
-  <draw:color draw:name="Red Hat 4" draw:color="#abb400"/>
-  <draw:color draw:name="Red Hat 5" draw:color="#4e376b"/>' && \
- tail -n 1 extras/source/palettes/standard.soc) > redhat.soc
-mv -f redhat.soc extras/source/palettes/standard.soc
-git commit -q -a -m 'add Red Hat colors to palette'
-
 # apply patches
 %autopatch -M 99
 %if 0%{?rhel}
-- 
2.43.5

